src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  padding: 11px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav__list-item + .nav__list-item {
+  margin-left: 20px;
+}
+
+.nav__list-link {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  padding: 23px 0;
```

Comment:
Paddings for inline elements don't works the way we expect, make it block or flex

Reply:
@OleksiiBlezniuk I don't understand why padding for inline elements doesn't work for this one. Could you explain, please?

Reply:
It's just a specifics of blocks model, they work different way. For blocks and inline blocks paddings increases element size and increase clickable zone, but for inline elements paddings just increases clickable zone, and can be collapsed

Reply:
@OleksiiBlezniuk understood, thx :)